openmls/tests/book_code.rs

```diff
@@ -271,10 +271,14 @@ fn book_operations(ciphersuite: Ciphersuite, provider: &impl OpenMlsProvider) {
         .build();
     // ANCHOR_END: mls_group_config_example
 
+    let welcome: MlsMessageIn = welcome.into();
```

Comment:
Maybe we should do an example of a `Client` that has to handle all the different message types. This would be nice for people to have an example how to use it. And show us if the way it works right now makes sense or if we should change something.

Reply:
Good point. I'll work on that in a separate PR.